index.html

```diff
@@ -1832,55 +1832,125 @@ <h2>Proof</h2>
     <h1>Core Representations</h1>
 
       <p>
-A <a>DID document</a> MUST be a single JSON object conforming to [[!RFC8259]].
-Many of the concepts in this document were introduced by example using the
-JSON-LD syntax, a format for mapping JSON data into the RDF semantic graph
-model, as defined by [[!JSON-LD]]. This section formalizes how the data model
-(described in Sections <a href="#data-model"></a> and
-<a href="#did-documents"></a>) is realized in JSON-LD.
-      </p>
-      <p>
-Although syntactic mappings are provided for JSON and JSON-LD only, applications
-and services can use any other data representation syntax, such as JXD (JSON XDI
-Data, a serialization format for the
-<a href="http://docs.oasis-open.org/xdi/xdi-core/v1.0/csd01/xdi-core-v1.0-csd01.xml">XDI graph model</a>),
+All concrete representations of a <a>DID document</a> MUST be serialized using a 
+deterministic mapping that is unambiguously able to be parsed into the data model
+defined in this specification. All serialization methods MUST define rules for 
+the bidirectional translation of a <a>DID document</a> both into and out of the
+representation in question. As a consequence, translation between any two formats
+MUST be done by first parsing the source format into a <a>DID document</a> model
+(described in Sections <a href="#data-model"></a> and <a href="#did-documents"></a>)
+and then serializing the <a>DID document</a> back out into the target format.
+      </p>
+
+      <p>
+Although syntactic mappings are provided for JSON and JSON-LD here, applications
+and services MAY use any other data representation syntax, such as
 XML, YAML, or CBOR, that is capable of expressing the data model.
       </p>
+      
+      <p>
+The production and consumption rules in this section apply to all implementations
+seeking to be fully compatible with independent implementations of the specification.
+Deployments of this specification MAY use a custom agreed-upon representation.
+See section ?? on Extensibility for more information.
+      </p>
 
     <section>
       <h2>
-Pure JSON
+JSON
       </h2>
 
+<section>
+      <h3>Production</h3>
+
         <p>
-The data model, as described in Section <a href="#data-model"></a>, can be
-encoded in Javascript Object Notation (JSON) [[RFC8259]] by mapping property
+A <a>DID document</a> MUST be a single JSON object conforming to [[!RFC8259]]. All
+top-level properties of the <a>DID document</a> MUST be represented by using the
+property name as the name of the member of the JSON object. The values of
```

Comment:
We need a way to refer to the key and the value of each thing in the DID document. The best term I could find from the document model section was "property", where the "name" of the property is the string value listed in the document, and the "value" of the property is whatever it's designed to convey.

Reply:
Ah, ok, now I understand. JSON uses "name" and "value". Javascript uses "key" and "value". RDF uses "subject" "predicate/property" "value". Some of us with feet in all of those camps have been using "property" and "value" with some success because it maps semi-cleanly between various communities (JSON/Javascript/RDF). "property name" and "name of the member" crosses the streams. 

Let's just use "property" and "value" in the spec... the VC spec survived multiple reviews by doing things like "X property" and "Y property" and doing things like "The value of the X property...".

Reply:
Yes, but VC is much more JSON-LD specific. I think this needs an additional issue (noted below) that I'll raise separately. Thanks!